CONTRIBUTING.rst

```diff
@@ -0,0 +1,10 @@
+.. image:: https://jazzband.co/static/img/jazzband.svg
+   :target: https://jazzband.co/
+   :alt: Jazzband
```

Comment:
This file will be used by GitHub and Sphinx documentation (included from doc/dev.rst)